fpdf/enums.py

```diff
@@ -198,6 +198,25 @@ def coerce(cls, value):
             return cls.L
         return super(cls, cls).coerce(value)
 
+class AlignV(CoerciveEnum):
```

Comment:
```suggestion
class VAlign(CoerciveEnum):
```
We usually talk about "vertical alignement", so I guess the "V" should come first

Reply:
Yes, that is what my first intuition was as well

But then the current `Align` class

```
class Align(CoerciveEnum):
    "Defines how to render text in a cell"

    C = intern("CENTER")
    "Center text horizontally"

    X = intern("X_CENTER")
    "Center text horizontally around current x position"

    L = intern("LEFT")
    "Left-align text"

    R = intern("RIGHT")
    "Right-align text"

    J = intern("JUSTIFY")
    "Justify text"
```

 should be renamed to `HAlign`  to be consistent, but that may be a bit too much.

So as a practical work around I choose `AlignV` such that the suggestion would come up when looking at `Align`.
Which then makes it weird that in the named arguments I use `v_align = AlignV.T`....

Reply:
> should be renamed to HAlign to be consistent, but that may be a bit too much.

You could rename it `HAlign` but then define this for backward-compatibility:
```
Align = HAlign
```

Reply:
Hm yeah, maybe I'm overthinking this but something in the back of my mind seems to prefer AlignV and AlignH above VAling and HAlign.

Probably because the two alignments are very similar, so their names should be similar as and should be put next to eachother when sorted alphabetically, for example in the documentation.

Maybe I'm just getting old. I'll go with whatever you prefer.

Reply:
renamed to VAlign as requested